labop/primitive_execution.py

```diff
@@ -89,50 +90,27 @@ def resolve_value(v):
                 return resolved
 
 def input_parameter_map(inputs: List[labop.ParameterValue]):
-    map = {}
+    map = {input.parameter.lookup().property_value.name: [] for input in inputs}
     for input in inputs:
         i_parameter = input.parameter.lookup().property_value
         value = input.value.get_value()
-        map[i_parameter.name] = value
+        map[i_parameter.name].append(value)
+    map = {k:(v[0] if len(v) == 1 else v) for k, v in map.items()}
```

Comment:
This appears to unpack a list and then pack it back into a tuple, when a Parameter has only a singleton value.  Why is this necessary, why not just leave it a list?